extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -122,7 +130,6 @@ public void build(RecorderContext recorderContext, HibernateOrmRecorder recorder
             BuildProducer<FeatureBuildItem> feature,
             BuildProducer<PersistenceUnitDescriptorBuildItem> persistenceUnitDescriptorProducer,
             BuildProducer<SubstrateResourceBuildItem> resourceProducer,
-            BuildProducer<HotDeploymentWatchedFileBuildItem> hotDeploymentProducer,
```

Comment:
Yeah, I think we used to use it and it got broken.

Reply:
It great that we'll have a test for it now. Good job @gwenneg :-)